### PR TITLE
Add Keyhacks information to Exploits where applicable

### LINKED ISSUE #151: Add Keyhacks to some of our exploits
https://github.com/streaak/keyhacks

We have a bunch of API regex https://github.com/bee-san/pyWhat/blob/main/pywhat/Data/regex.json#L137

These exploits are similar to `keyhacks`

Go through our API keys (ctrl + F API) and add the appropriate Keyhacks command to them :)

### PR DESCRIPTION
**Pull request by vanjo9800**
Closes #151

### CODE REVIEW

File: pywhat/Data/regex.json [hunk 1]
```diff
@@ -201,6 +201,7 @@
       "Regex": "^(sandbox-?sq0csp-[0-9A-Za-z-_]{43}|sq0[a-z]{3}-[0-9A-Za-z-_]{22,43})$",
       "plural_name": false,
       "Description": null,
+      "Exploit": "Use the command below to verify that the secret is valid:\n  $ curl \"https://squareup.com/oauth2/revoke\" -d '{\"access_token\":\"RANDOM_STRING\",\"client_id\":\"APP_ID\"}'  -H \"Content-Type: application/json\" -H \"Authorization: Client SECRET_HERE\"\n  API Documentation: https://developer.squareup.com/us/en",
```
Comment (piatrashkakanstantinass) [inline pywhat/Data/regex.json:204]:
```suggestion
      "Exploit": "Use the command below to verify that the secret is valid:\n  $ curl \"https://squareup.com/oauth2/revoke\" -d '{\"access_token\":\"[RANDOM_STRING]\",\"client_id\":\"[APP_ID]\"}'  -H \"Content-Type: application/json\" -H \"Authorization: Client SECRET_HERE\"\n",
```

**Review by piatrashkakanstantinass — CHANGES_REQUESTED** [review]
Please remove `API Documentation:` at the end of exploits and make sure that every placeholder is in `XXXX_HERE` format (`AUTH_TOKEN` should be `AUTH_TOKEN_HERE`)

File: pywhat/Data/regex.json [hunk 2]
```diff
@@ -284,7 +288,7 @@
       "Description": null,
       "Exploit": null,
       "Rarity": 1,
-      "URL": null,
+      "URL": "https://graph.facebook.com/oauth/access_token?client_id=ID_HERE&client_secret=SECRET_HERE&redirect_uri=&grant_type=client_credentials",
```
Comment (bee-san) [inline pywhat/Data/regex.json:291]: Hey 👋 This isn't how URLs work in our program :) Per our docs:

> URL (string): If there is a way to analyse the text online, put it here. An example is Latitude & Longitude coordinates with a URL linking to Google Maps. So it'd do maps.google.com/place/. This way it takes you directly there. Not all regex have URls.

So let's say it matches on "TEST_STRING_HERE"

It'll print:

https://graph.facebook.com/oauth/access_token?client_id=ID_HERE&client_secret=SECRET_HERE&redirect_uri=&grant_type=client_credentialsTEST_STRING_HERE

Instead, you can use the `Exploit:`  key. And where it says `SECRET_HERE` replace with `AUTH_TOKEN_HERE` :)

**Review by bee-san — CHANGES_REQUESTED** [review]

File: pywhat/Data/regex.json (hunk 2 above)
Comment (vanjo9800) [reply pywhat/Data/regex.json]: Yeah, got it!

Replaced now and tested with `fixtures/file`.

**Comment by bee-san** [comment]
> Please remove API Documentation:

Why?

**Review by bee-san — APPROVED** [review]

**Review by bee-san — APPROVED** [review]

**Pull request merged**